src/ripple/overlay/impl/ProtocolMessage.h

```diff
@@ -141,7 +142,11 @@ parseMessageHeader(
     MessageHeader hdr;
     auto iter = buffersBegin(bufs);
```

Comment:
Check for an empty buffer sequence?

Reply:
Since that's a precondition of this function, I'd document that as part of the docstring. You could also consider adding an `assert`

Reply:
Done